src/components/utils/include/utils/logger/logger_impl.h

```diff
@@ -74,10 +74,14 @@ class LoggerImpl : public Logger, public LoggerInitializer {
   bool IsEnabledFor(const std::string& component,
                     LogLevel log_level) const override;
   void PushLog(const LogMessage& log_message) override;
+  void InitLoggerSettings(LoggerSettings* settings) override;
+  void InitFlushLogsTimePoint(const TimePoint& time_point) override;
 
   std::unique_ptr<ThirdPartyLoggerInterface> impl_;
+  LoggerSettings* settings_;
   LoopThreadPtr<LogMessageLoopThread> loop_thread_;
   bool use_message_loop_thread_;
+  TimePoint flush_logs_time_point_;
```

Comment:
what's the default value of flush_logs_time_point_ if it is not initialized?

Reply:
Default constructor, creates a time_point representing the Clock's epoch (i.e., time_since_epoch() is zero).

Reply:
@VladSemenyuk Got it. Thank you